src/sentry/integrations/discord/utils/consts.py

```diff
@@ -0,0 +1,249 @@
+    50161: "Voice messages must have supporting metadata.",
+    50162: "Voice messages cannot be edited.",
+    50163: "Cannot delete guild subscription integration",
+    50173: "You cannot send voice messages in this channel.",
+    50178: "The user account must first be verified",
+    50600: "You do not have permission to send this sticker.",
+    60003: "Two factor is required for this operation",
+    80004: "No users with DiscordTag exist",
+    90001: "Reaction was blocked",
+    90002: "User cannot use burst reactions",
+    110001: "Application not yet available. Try again later",
+    130000: "API resource is currently overloaded. Try again a little later",
+    150006: "The Stage is already open",
+    160002: "Cannot reply without permission to read message history",
+    160004: "A thread has already been created for this message",
+    160005: "Thread is locked",
+    160006: "Maximum number of active threads reached",
+    160007: "Maximum number of active announcement threads reached",
+    170001: "Invalid JSON for uploaded Lottie file",
+    170002: "Uploaded Lotties cannot contain rasterized images such as PNG or JPEG",
+    170003: "Sticker maximum framerate exceeded",
+    170004: "Sticker frame count exceeds maximum of 1000 frames",
+    170005: "Lottie animation maximum dimensions exceeded",
+    170006: "Sticker frame rate is either too small or too large",
+    170007: "Sticker animation duration exceeds maximum of 5 seconds",
+    180000: "Cannot update a finished event",
+    180002: "Failed to create stage needed for stage event",
+    200000: "Message was blocked by automatic moderation",
+    200001: "Title was blocked by automatic moderation",
+    220001: "Webhooks posted to forum channels must have a thread_name or thread_id",
+    220002: "Webhooks posted to forum channels cannot have both a thread_name and thread_id",
+    220003: "Webhooks can only create threads in forum channels",
+    220004: "Webhook services cannot be used in forum channels",
+    240000: "Message blocked by harmful links filter",
+    350000: "Cannot enable onboarding, requirements are not met",
+    350001: "Cannot update onboarding while below requirements",
+}
+
+"""
+After an alert rule is created, the user can perform actions that can break the alert and
+produce error Discord API responses.
+
+For our SLO that monitors Discord response status codes, we _do_ want to keep track of
+responses that indicate a problem from our end i.e. 50109
+(the request body contains invalid JSON.).
+We _don't_ want to keep track of responses that are failing because of a user config change.
+
+In order to make the SLO more relevant, we want to ignore specific discord provided codes that
+can be returned when a user changes a configuration and breaks the alert.
+
+Complete list of codes: https://discord.com/developers/docs/topics/opcodes-and-status-codes#http
+
+Here is a working list of codes that we should ignore and why:
```

Comment:
I would add a note here saying that we're only doing this until we can bubble this up to the user and help them fix their broken integration. but for now we don't have anything actionable so we are logging it